README.md

```diff
@@ -1,2 +1,105 @@
 # SeineSailor
-Help developers navigating through the codebase smoothly
+
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+[![GitHub](https://img.shields.io/github/last-commit/SeineAI/SeineSailor/main?style=flat-square)](https://github.com/SeineAI/SeineSailor/commits/main)
+
+## Overview
+
+SeineSailor is an AI-powered copilot for project managers, designed to assist with code reviews, project discussions,
+and knowledge transfer. It leverages the power of WatsonX's `llama3` and `mixt-8x7B` models to provide intelligent and
+context-aware support throughout the development lifecycle.
+
+## Features
+
```

Comment:
test to see if @SeineSailor works in commit_comments

Reply:
looks like "starting a review" didn't trigger.

Reply:
try to see if "add review comment" works @SeineSailor

Reply:
@SeineSailor maybe position matters?

Reply:
added   pull_request_review:
    types: [submitted, edited]
    
 @SeineSailor test

Reply:
still didn't trigger